src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java

```diff
@@ -40,12 +40,19 @@ public void verifyNoSlotPresent(String slotName) {
   }
 
   public void verifySlotExpanded(String slotName) {
+    By slotById = By.id(slotName);
+    WebElement slot = driver.findElement(slotById);
+
+    wait.forElementVisible(slotById);
     scrollToSlotOnMobile(slotName);
-    WebElement slot = driver.findElement(By.id(slotName));
+
     if (checkIfSlotExpanded(slot)) {
-      Log.log("AdInSlot", "Slot expanded as expecting", true);
+      Log.log("AdInSlot",
+              "Slot (" + slotName + ") expanded as expecting",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Slot (" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)